sdk/python/examples/bayesianoptimization-katib-sdk.ipynb

```diff
@@ -0,0 +1,575 @@
+    "from kubeflow.katib import V1alpha3TrialTemplate"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 28,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "algorithmsettings = V1alpha3AlgorithmSetting(\n",
+    "    name= \"random_state\",\n",
+    "    value = \"10\"\n",
+    "    )\n",
+    "algorithm = V1alpha3AlgorithmSpec(\n",
+    "    algorithm_name = \"bayesianoptimization\",\n",
+    "    algorithm_settings = [algorithmsettings]\n",
+    "  )\n",
+    "\n",
+    "# Metric Collector\n",
+    "collector = V1alpha3CollectorSpec(kind = \"TensorFlowEvent\")\n",
+    "FileSystemPath = V1alpha3FileSystemPath(kind = \"/train\" , path = \"Directory\")\n",
+    "metrics_collector_spec = V1alpha3MetricsCollectorSpec(\n",
+    "    collector = collector,\n",
+    "    source = FileSystemPath)\n",
+    "\n",
+    "# Objective\n",
+    "objective = V1alpha3ObjectiveSpec(\n",
+    "    goal = 0.9999,\n",
+    "    objective_metric_name = \"Validation-accuracy\",\n",
+    "    type = \"maximize\")\n",
+    "\n",
+    "# Parameters\n",
+    "\n",
+    "feasible_space_optimizer = V1alpha3FeasibleSpace(list = [\"sgd\", \"adam\", \"ftrl\"])\n",
+    "feasible_space_lr = V1alpha3FeasibleSpace(min = \"0.01\", max = \"0.03\")\n",
+    "feasible_space_layers = V1alpha3FeasibleSpace(min = \"2\", max = \"5\")\n",
+    "\n",
+    "parameters = [V1alpha3ParameterSpec(\n",
+    "    feasible_space = feasible_space_optimizer, \n",
+    "    name = \"--optimizer\",\n",
+    "    parameter_type = \"categorical\"\n",
+    "    ),\n",
+    "    V1alpha3ParameterSpec(\n",
+    "    feasible_space = feasible_space_lr, \n",
+    "    name = \"--lr\",\n",
+    "    parameter_type =\"double\"\n",
+    "    ),\n",
+    "    V1alpha3ParameterSpec(\n",
+    "    feasible_space = feasible_space_layers, \n",
+    "    name = \"--num-layers\",\n",
+    "    parameter_type = \"int\"\n",
+    "    )]\n",
+    "\n",
+    "# Trialtemplate\n",
+    "go_template = V1alpha3GoTemplate(\n",
+    "    raw_template =   \"apiVersion: \\\"batch/v1\\\"\\nkind: Job\\nmetadata:\\n  name: {{.Trial}}\\n  namespace: {{.NameSpace}}\\nspec:\\n  template:\\n    spec:\\n      containers:\\n      - name: {{.Trial}}\\n        image: docker.io/kubeflowkatib/mxnet-mnist\\n        command:\\n        - \\\"python3\\\"\\n        - \\\"/opt/mxnet-mnist/mnist.py\\\"\\n        - \\\"--batch-size=64\\\"\\n        {{- with .HyperParameters}}\\n        {{- range .}}\\n        - \\\"{{.Name}}={{.Value}}\\\"\\n        {{- end}}\\n        {{- end}}\\n      restartPolicy: Never\"\n",
+    "    )\n",
+    "\n",
+    "\n",
+    "trial_template= V1alpha3TrialTemplate(go_template=go_template)\n",
+    "\n",
+    "\n",
+    "# Experiment\n",
+    "experiment = V1alpha3Experiment(\n",
+    "    api_version=\"kubeflow.org/v1alpha3\",\n",
+    "    kind=\"Experiment\",\n",
+    "    metadata=V1ObjectMeta(name=\"bayesianoptimization\",namespace=\"anonymous\"),\n",
```

Comment:
Is `anonymous` namespace here correct? I can see bellow, that you are using `kube-prem` namespace.

Reply:
changes updated